manage.py

```diff
@@ -1,8 +1,11 @@
+from configobj import ConfigObj
 from flask_script import Manager, Server, Shell
 
 from profiles.factory import create_app
 
-app = create_app()
+config = ConfigObj('app.cfg')
```

Comment:
`.gitignore` should ignore this I think. The formula provides a file to be expanded: https://github.com/elifesciences/profiles-formula/blob/master/salt/profiles/config/srv-profiles-app.cfg

Reply:
Yep, it was already there.